android/build.gradle

```diff
@@ -192,3 +192,58 @@ dependencies {
         }
     }
 }
+
+def isScreensExampleApp() {
+    return project.hasProperty('isScreensExampleApp') && project.property('isScreensExampleApp')
+}
+
+task copyCodegenArtifacts(type: Copy) {
+    group 'After build tasks'
+    description 'Tasks which copy codegen artifacts to paper architecture'
+    
+    if (!isScreensExampleApp()) {
+        return
+    }
+
+    dependsOn tasks.generateCodegenArtifactsFromSchema
+
+    def absoluteCodegenArtifactsPaperDestination = "${project.rootDir}/../../${project.property('codegenArtifactsPaperDestination')}"
+    def absoluteCodegenArtifactsSource = "${project.rootDir}/../../${project.property('codegenArtifactsSource')}"
+
+    def existingFiles = fileTree(absoluteCodegenArtifactsPaperDestination).matching {
+        include '**/*.java'
+    }
+
+    def generatedFiles = fileTree(absoluteCodegenArtifactsSource).matching {
+        include '**/*.java'
+    }
+
+    def existingFilesMap = [:]
+
+    existingFiles.forEach { existingFile ->
+        existingFilesMap[existingFile.name] = 1
+    }
+
+    generatedFiles.forEach { generatedFile -> 
+        if (!existingFilesMap.containsKey(generatedFile.name)) {
+            println generatedFile.name + " not found in paper dir"
```

Comment:
Also I do not remember, but it would be worth checking whether there is a `warn` function of something, it seems that it would be appropriate to use it here.

Reply:
Tried logger.warn but it behaves the same. So I stick to println.

Reply:
I doubt it behaves the same. It maybe looks the same, but it has different log level associated - WARN - instead of QUIET, I believe, for `println`.

Is is important? Definitely no. If you want to really stay with current approach it's ok.

Reply:
You're right. Changed in https://github.com/software-mansion/react-native-screens/pull/2168/commits/5afd502c2b94a6035ac128860340695feaffc4e6